src/aims_dev_ui/config/dev.py

```diff
@@ -4,8 +4,9 @@
 
 SECRET_KEY = 'secretkey'
 
-API_URL = os.getenv('API_URL') or 'http://localhost:9000'
+#API_URL = os.getenv('API_URL') or 'http://localhost:9000'
+API_URL = os.getenv('API_URL') or 'https://initial-test-bulk-1.aims.gcp.onsdigital.uk'
```

Comment:
And here....

Reply:
I have created a separate ticket to parameterise the API URL as part of the concourse job. For now this is the only URL to use.

Reply:
The ENV variable will be replaced during deploy in the terraform.
I also don't think including the URL in the repo is good practice either.